crates/red_knot_python_semantic/resources/mdtest/import/star.md

```diff
@@ -549,26 +559,54 @@ from a import *
 
 reveal_type(X)  # revealed: bool
 
-# TODO: should emit error: [unresolved-reference]
+# error: [unresolved-reference]
 reveal_type(Y)  # revealed: Unknown
 
-# TODO: The `*` import should not be considered a redefinition
+# The `*` import should not be considered a redefinition
 # of the global variable in this module, as the symbol in
 # the `a` module is in a branch that is statically known
 # to be dead code given the `python-version` configuration.
-# Thus this should reveal `Literal[True]`.
-reveal_type(Z)  # revealed: Unknown
+# Thus this correctly reveals `Literal[True]` and does not have an error emitted
+reveal_type(Z)  # revealed: Literal[True]
 
 if sys.version_info >= (3, 12):
     from aa import *
 
-    # TODO: should reveal `Never`
+    # TODO: should not error because this branch is unreachable;
+    # should reveal `Never` (see https://github.com/astral-sh/ruff/issues/15967)
```

Comment:
I think you probably want to reference https://github.com/astral-sh/ruff/issues/15797 here? In fact, this error might be gone if you rebase on main?

Reply:
> I think you probably want to reference https://github.com/astral-sh/ruff/issues/15797 here?

oops, yes, thanks!

> In fact, this error might be gone if you rebase on main?

Indeed it is! It looks like we still infer `Unknown`, though. That might be okay? Not sure?

Reply:
> Indeed it is! It looks like we still infer `Unknown` rather than `Never`, though. That might be okay? Not sure -- WDYT?

Yes, this is expected.